src/components/timeline/TrackScreenshots.js

```diff
@@ -158,7 +156,7 @@ class Screenshots extends PureComponent<Props, State> {
         style={{ height: trackHeight }}
         onMouseLeave={this._handleMouseLeave}
         onMouseMove={this._handleMouseMove}
-        onMouseUp={this._selectScreenshotOnClick}
+        onClick={this._selectScreenshotOnClick}
```

Comment:
Tests were failing when I wouldn't do that, but when I tested this looked fine even without this change. But I still made this change because I thought this could fail in subtle cases.